spec/ssm_config_activerecord_spec.rb

```diff
@@ -1,36 +1,99 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-
+require 'timecop'
 class SsmConfigDummy < ::ActiveRecord::Base
   self.table_name = 'ssm_config_records'
 end
 
 RSpec.describe SsmConfig do
   before do
-    stub_const('SsmConfig::CONFIG_PATH', '../fixtures')
-    stub_const('SsmConfig::ACTIVE_RECORD_MODEL', 'SsmConfigDummy')
+    stub_const('SsmStorageFile::CONFIG_PATH', '../fixtures')
+    stub_const('SsmStorageDb::ACTIVE_RECORD_MODEL', 'SsmConfigDummy')
     run_migrations(:up, migrations_path, 1)
+    SsmConfigDummy.new(:file => 'data', :accessor_keys => 'test,[0]', :value => 'hello').save
+    SsmConfigDummy.new(:file => 'data', :accessor_keys => 'other_key', :value => 'goodbye').save
+    Timecop.freeze(Time.zone.now)
```

Comment:
should we freeze time before we create the models? Then we can travel to the new time for updates? I wonder we should only travel in the updates spec to clearly define what is happening? Add the `travel` as a before on the context on 'making changes'.